BioForm/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.permissions import BasePermission
+
+
+class IsJWTAuthenticated(BasePermission):
+    message = 'You must be logged in.'
+
+    def has_permission(self, request, view):
+        return request.api_user is not None
+        
+
+class IsUsersObject(IsJWTAuthenticated):
+
+    def has_object_permission(self, request, view, obj):
+        return getattr(obj, 'user_id') == getattr(request.api_user, 'id')
```

Comment:
Here make sure `request.api_user` is not `None` else `getattr(request.api_user, 'id')` will throw an error.

Reply:
change made as suggested